diff -up libical-4.0.4/src/libical/libical.pc.in.omv~ libical-4.0.4/src/libical/libical.pc.in
--- libical-4.0.4/src/libical/libical.pc.in.omv~	2026-07-25 13:22:56.000000000 +0200
+++ libical-4.0.4/src/libical/libical.pc.in	2026-08-02 20:50:48.579208109 +0200
@@ -9,5 +9,4 @@ Name: libical
 Description: @CMAKE_PROJECT_DESCRIPTION@
 URL: @CMAKE_PROJECT_HOMEPAGE_URL@
 Version: @CMAKE_PROJECT_VERSION@
-Libs: -L${libdir} -lical@PTHREAD_LIBS_PRIVATE@@REQUIRES_PRIVATE_ICU@
-Cflags: -I${includedir}
+Libs: -lical@PTHREAD_LIBS_PRIVATE@@REQUIRES_PRIVATE_ICU@
diff -up libical-4.0.4/src/libical-glib/libical-glib.pc.in.omv~ libical-4.0.4/src/libical-glib/libical-glib.pc.in
--- libical-4.0.4/src/libical-glib/libical-glib.pc.in.omv~	2026-07-25 13:22:56.000000000 +0200
+++ libical-4.0.4/src/libical-glib/libical-glib.pc.in	2026-08-02 20:50:48.595489368 +0200
@@ -2,13 +2,9 @@
 # SPDX-License-Identifier: LGPL-2.1-only OR MPL-2.0
 
 prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
 
 Name: libical-glib
 Description: A GObject interface of the libical library
 Version: @CMAKE_PROJECT_VERSION@
 Requires: glib-2.0, gobject-2.0, libical
-Cflags: -I${includedir}
-Libs: -L${libdir} -lical-glib
+Libs: -lical-glib
