diff -uraN libical-4.0.4/CMakeLists.txt omv-libical-4.0.4/CMakeLists.txt
--- libical-4.0.4/CMakeLists.txt	2026-07-25 13:22:56.000000000 +0200
+++ omv-libical-4.0.4/CMakeLists.txt	2026-08-02 20:51:02.192310578 +0200
@@ -785,9 +785,9 @@
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_XOPEN_SOURCE=500 -D_DEFAULT_SOURCE -D_GNU_SOURCE")
   endif()
 endif()
-if(CMAKE_C_COMPILER_IS_CLANG)
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Qunused-arguments")
-endif()
+#if(CMAKE_C_COMPILER_IS_CLANG)
+#  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Qunused-arguments")
+#endif()
 
 set(CMAKE_CXX_COMPILER_IS_CLANG False)
 set(CMAKE_CXX_COMPILER_IS_GCC False)
