diff -up libical-3.0.8/src/libical/icalrecur.c.omv~ libical-3.0.8/src/libical/icalrecur.c
--- libical-3.0.8/src/libical/icalrecur.c.omv~	2020-11-10 22:17:39.654648325 +0100
+++ libical-3.0.8/src/libical/icalrecur.c	2020-11-10 22:17:46.308708710 +0100
@@ -1018,7 +1018,7 @@ icalarray *icalrecurrencetype_rscale_sup
 
     calendars = icalarray_new(sizeof(const char **), 20);
 
-    en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
+    en = ucal_getKeywordValuesForLocale("calendar", NULL, false, &status);
     while ((cal = uenum_next(en, NULL, &status))) {
         cal = icalmemory_tmp_copy(cal);
         icalarray_append(calendars, &cal);
@@ -1411,7 +1411,7 @@ static int initialize_rscale(icalrecur_i
         }
 
         /* Check if specified calendar is supported */
-        en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
+        en = ucal_getKeywordValuesForLocale("calendar", NULL, false, &status);
         while ((cal = uenum_next(en, NULL, &status))) {
             if (!strcmp(cal, rule.rscale)) {
                 is_hebrew = !strcmp(rule.rscale, "hebrew");
